Examples/Tests/point_of_contact_EB/analysis.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
```

Comment:
Could you add a brief description of what the simulation is doing, and of what this test is then checking?